rnd/infra/terraform/environments/dev.tfvars

```diff
@@ -1,11 +1,12 @@
-project_id     = "agpt-dev"
-region         = "us-central1"
-zone           = "us-central1-a"
-network_name   = "dev-gke-network"
-subnet_name    = "dev-gke-subnet"
-subnet_cidr    = "10.0.0.0/24"
-cluster_name   = "dev-gke-cluster"
-node_count     = 2
-node_pool_name = "dev-main-pool"
-machine_type   = "e2-medium"
-disk_size_gb   = 100
+project_id      = "agpt-dev"
```

Comment:
It would be better if we used a formatting that didn't require such a large diff when adding a new variable.

It's fine whilst it's a small file. But will make things harder to review in the future if we end up with many variables

Reply:
what would you prefer?

Reply:
@Swiftyos im gonna resolve this for now - but can defs update formatting, lmk